hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterAdminCLI.java

```diff
@@ -78,6 +81,8 @@
   private static RouterClient client;
   private static Router router;
 
+  private static DistributedFileSystem hdfs;
```

Comment:
I don't think we need any HDFS feature do we? Let's do: fs = cluster.getCluster().getFileSystem();